### ISSUE TITLE
[@thi.ng/base-n] bug when converting some values

### ISSUE THREAD

**Issue by ZYinMD** [thi-ng/umbrella#308]
v1.0.4

```js
import { BASE62 } from "@thi.ng/base-n";

function numberToBase62AndBack(original) {
  const converted = BASE62.encodeBigInt(BigInt(original));
  const reversed = BASE62.decodeBigInt(converted).toString(10);
  console.log(reversed === original);
}

numberToBase62AndBack("42"); // true
numberToBase62AndBack("123456"); // true
numberToBase62AndBack("1629787729186"); // false ?!?!?!?!?!?!?!?!
numberToBase62AndBack("6978223673954094342"); // true
numberToBase62AndBack("2324234239736739235357657"); // true

```
This is really surprising. I don't know why.

**Comment by postspectacular** [comment]
Thank you, @ZYinMD for reporting this issue! The weirdness was caused by an unintentional integer cast of intermediate values... will release new version later today!

**Comment by postspectacular** [comment]
So that's released as v1.0.5 :) Thanks again - good find!
